hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineModule.java

```diff
@@ -440,18 +439,18 @@ private JsonClient jsonClient(JsonResponseProcessor jsonResponseProcessor, Envir
 
     @Provides
     @Singleton
-    private IdaKeyStore getKeyStore(X509CertificateFactory certificateFactory, SamlEngineConfiguration configuration) {
+    private IdaKeyStore getKeyStore(SamlEngineConfiguration configuration) {
```

Comment:
So.. tell me about guice and what the impact of this is

Reply:
there isn't one :D